.github/workflows/experimental-build.yml

```
@@ -1,7 +1,7 @@
 ---
 name: experimental-build
 
-on: [push]
+on: [push, pull_request]
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)